doc/developer_guide/developer_guide.md

```diff
@@ -1,8 +1,66 @@
 # Developer Guide
 
-
-In this developer guide we explain how you can build this project.
+The developer guide explains how to maintain and develop the Advanced Analytics Framework (AAF).
 
 * [developer_environment](developer_environment.md)
-* [building_documentation](building_documentation.md)
 
+## Building and Installing the AAF Script Language Container (SLC)
+
+The following command builds the SLC for the AAF
+
+```shell
+poetry run nox -s build_language_container
+```
+
+Installing the SLC ins described in the [AAF User Guide](../user_guide/user_guide.md#script-language-container-slc).
+
+## Running Tests
+
+AAF comes with different automated tests implemented in different programming languages and requiring different environments:
+
+| Language | Category                        | Database | Environment |
+|----------|---------------------------------|----------|-------------|
+| Python   | Unit tests                      | no       | poetry      |
```

Comment:
are you sure, this might also test the lua script generation and this needs amalg.